### DISCUSSION TITLE
Support Needed: Configuring RustDesk for Multiple Users on macOS

### DISCUSSION THREAD

**Discussion by EC5987** [rustdesk/rustdesk-server-pro#380] [General]
I'm using a pro licensed self-hosted RustDesk server for my team to access multiple machines, and we've encountered an issue. Each of my colleagues has a separate user account on every computer, deployed via JumpCloud. However, whenever we try to switch users, we lose the connection to that computer via RustDesk.

I've been unable to find a solution to this problem. Is there a way to configure RustDesk to support multiple users on macOS without losing the connection when switching accounts?

Any guidance or support would be greatly appreciated!

**Comment by rustdesk** [comment]
Not possible yet.
